lib/edifunct/tokenizer.rb

```diff
@@ -6,9 +6,13 @@ module Edifunct
   # Tokenizer is responsible for splitting message into segments, data elements and components.
   class Tokenizer
     class << self
-      def for_message(_edifact_message)
-        # TODO: Should check if the message starts with `UNA`, and then extract the different separator/terminator settings to be used for initializing the tokenizer.
-        new
+      def for_message(edifact_message)
+        if edifact_message =~ /\AUNA/
```

Comment:
```suggestion
        if edifact_message[0..2] == 'UNA'
```

Reply:
why the second one is better than the first ?? 🤔

Reply:
Regex seems like overkill for something like this.